spec/features/user_follows_a_person_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+feature 'User following' do
+  scenario 'user follows and unfollows' do
+    amy = Fabricate(:user)
+    category = Fabricate(:category)
+    video = Fabricate(:video, category: category)
+    Fabricate(:review, user: amy, video: video)
+
+    sign_in
+    click_on_video_on_home_page(video)
+    click_link amy.full_name
+    click_link "Follow"
+
+    expect(page).to have_content(amy.full_name)
+    unfollow(amy)
+    expect(page).not_to have_content(amy.full_name)
+  end  
+end
+
+def unfollow(user)
+  find("a[data-method='delete']").click
```

Comment:
This does not use the user that is provided. If you are not using the parameter, you don't need to require it.